repos/spack_repo/builtin/packages/curl/package.py

```diff
@@ -181,9 +181,12 @@ class Curl(NMakePackage, AutotoolsPackage, CMakePackage):
     # https://github.com/curl/curl/pull/9054
     patch("easy-lock-sched-header.patch", when="@7.84.0")
 
-    platform_default = "cmake" if sys.platform =="win32" else "autotools"
+    platform_default = "cmake" if sys.platform == "win32" else "autotools"
```

Comment:
this defines a class attribute, I would inline it

Reply:
Updated.